tests/test_react_auth.py

```diff
@@ -0,0 +1,77 @@
+from foursight_core.react.api.jwt_utils import jwt_decode
+from test_react_auth_defs import *
```

Comment:
`*` isn't necessarily the best since it can make it hard to track down usages and side effects. I would consider fixture-izing what is in that file and importing explicitly here so things can be easily traced.

Reply:
All set (import individually).